server/controllers/entities/lib/resolver/update_resolved_entry.js

```diff
@@ -34,10 +34,29 @@ const getEntity = (prefix, entityId) => {
   }
 }
 
-const addMissingClaims = (seedClaims, userId, batchId) => entity => {
-  // Do not update if property already exists
+const updateClaims = (seedClaims, userId, batchId, forceUpdateProps) => entity => {
+  addMissinClaims(seedClaims, userId, batchId, entity)
+  // Only update if property exists in forceUpdateProps
   // Known cases: avoid updating authors who are actually edition translators
+  if (forceUpdateProps && _.some(forceUpdateProps)) {
+    forceUpdateClaims(seedClaims, userId, batchId, forceUpdateProps, entity)
+  }
+}
+const forceUpdateClaims = (seedClaims, userId, batchId, forceUpdateProps, currentDoc) => {
+  const updatedDoc = _.cloneDeep(currentDoc)
+  const { claims } = updatedDoc
+  forceUpdateProps.forEach(prop => {
+    if (claims && claims[prop]) {
+      claims[prop] = seedClaims[prop]
+    }
+  })
+  if (!_.isEqual(claims, currentDoc.claims)) {
+    entities_.putUpdate({ userId, currentDoc, updatedDoc })
```

Comment:
by calling `entities_.putUpdate` directly, it skips `validateClaims` (`validateAndFormatClaims` on current master branch): that's a problem, as mistakes happens, and this validation can be precious to avoid a wide range of errors (ex: setting an entity of type `work` in place of a `publisher` or setting an edition ISBN to an ISBN already used by another edition)